WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Dynamic/DynamicDashboardCard.swift

```diff
@@ -141,31 +141,31 @@ extension DSButtonStyle {
     }
 }
 
-#if DEBUG
-struct DynamicDashboardCard_Previews: PreviewProvider {
-    static var previews: some View {
-        DynamicDashboardCard(
-            input: .init(
-                featureImageURL: URL(string: "https://i.pickadummy.com/index.php?imgsize=400x200")!,
-                featureImageWidthToHeightRatio: 2,
-                rows: [
-                    .init(
-                        title: "Title first",
-                        description: "Description first",
-                        imageURL: URL(string: "https://mobiledotblog.files.wordpress.com/2024/03/perf-icon.png")!
-                    ),
-                    .init(
-                        title: "Title second",
-                        description: "Description second",
-                        imageURL: nil
-                    )
-                ],
-                action: .init(title: "Action button", callback: {
-                    ()
-                })
-            )
-        )
-        .padding()
-    }
-}
-#endif
+//#if DEBUG
+//struct DynamicDashboardCard_Previews: PreviewProvider {
+//    static var previews: some View {
+//        DynamicDashboardCard(
+//            input: .init(
+//                featureImageURL: URL(string: "https://i.pickadummy.com/index.php?imgsize=400x200")!,
+//                featureImageWidthToHeightRatio: 2,
+//                rows: [
+//                    .init(
+//                        title: "Title first",
+//                        description: "Description first",
+//                        imageURL: URL(string: "https://mobiledotblog.files.wordpress.com/2024/03/perf-icon.png")!
+//                    ),
+//                    .init(
+//                        title: "Title second",
+//                        description: "Description second",
+//                        imageURL: nil
+//                    )
+//                ],
+//                action: .init(title: "Action button", callback: {
+//                    ()
+//                })
+//            )
+//        )
+//        .padding()
+//    }
+//}
+//#endif
```

Comment:
For some reason, this code breaks the SwiftUI Previews even when the "Design System" target is selected.

Reply:
Do you mean it breaks all the other previews too? I uncommented it, and this particular preview really doesn't work, but the `ContentPreview` works perfectly. 🤔

Reply:
In my case I get this error when trying to preview the `ContentPreview` view:

![CleanShot 2024-04-24 at 17 09 27@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/76ed6e4c-7c62-43a5-8471-5c00e5bf1e89)

Reply:
Perhaps it's a local issue.

Reply:
Resolved the issue by deleting `DynamicDashboardCard.swift` file and adding it again 🤷‍♂️

Reply:
"Try to restart your computer"-like classic 😄